jupyterbook/content/code_gallery/data_access_notebooks/2017-03-21-ERDDAP_IOOS_Sensor_Map.ipynb

```diff
@@ -8,34 +8,20 @@
     "\n",
```

Comment:
<p>Spot on, but you can say it is because the sensor map only shows real-time observations (past 4-hours). </p><p> </p><blockquote>The Environmental Sensor Map integrates regional, national, and global real-time (past 4-hours) data across the IOOS enterprise.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ioos/ioos_code_lab/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I did not know that! I thought it was "recent" data but not ~4 hours.</p>

Reply:
Got it. I made the changes and it is ready for another round of reviews.